Packs/Workday/Integrations/WorkdayIAMEventsGenerator/README.md

```diff
@@ -0,0 +1,78 @@
+Generates Workday dummy test reports and events. Used for testing/dev purposes.
+
+To use this integration for testing:
+1. Create an instance - Add a listening port and mark the *Long running instance* checkbox.
```

Comment:
1. Create an instance.
   a. Add a listening port
   b. Select the **Long running instance** checkbox.